ark/utils/notebooks_test.py

```diff
@@ -23,3 +33,66 @@ def test_example_spatial_analysis():
 
 def test_example_neighborhood_analysis():
     _exec_notebook('example_neighborhood_analysis_script.ipynb')
+
+
+# testing specific inputs for Segment_Image_Data
+# test mibitiff, 6000 seconds = default timeout on Travis
+@testbook(SEGMENT_IMAGE_DATA_PATH, timeout=6000)
+def test_segment_image_data_mibitiff(tb):
+    with tdir() as tiff_dir, tdir() as input_dir, tdir() as output_dir, \
+         tdir() as single_cell_dir, tdir() as viz_dir:
+        # create input files
+        notebooks_test_utils.segment_notebook_setup(tb,
+                                                    deepcell_tiff_dir=tiff_dir,
+                                                    deepcell_input_dir=input_dir,
+                                                    deepcell_output_dir=output_dir,
+                                                    single_cell_dir=single_cell_dir,
+                                                    viz_dir=viz_dir,
+                                                    is_mibitiff=True)
+
+        # hard coded fov setting
+        notebooks_test_utils.fov_channel_input_set(
+            tb,
+            fovs_to_load=['fov0_otherinfo-MassCorrected-Filtered.tiff',
+                          'fov1-MassCorrected-Filtered.tiff'],
+            nucs_list=['chan0'],
+            mems_list=['chan1', 'chan2'])
+
+        # generate the deepcell output files from the server
+        tb.execute_cell('create_output')
```

Comment:
I think sending data to the server is leading to slow testing; this now accounts for the majority of the Travis build time. Can we replace this with a test function to perform the same thing? 
![image](https://user-images.githubusercontent.com/13770365/99032548-a6143e80-252e-11eb-8b4b-1aebc076a4b9.png)

Reply:
Yeah definitely agree. So basically a function that creates a dummy `segmentation_labels` value? Would be really easy to do.

Reply:
My assumption is that this is the slowest step; it could be something else in the notebook. Whatever it is, we should speed it up.

Reply:
Just realized that this method would also force us to bypass the segmentation label loading step. We'll need to re-create a random `_feature_0.tif` for each `fov` specified. Shouldn't be much harder though.